wiki/Tournaments/YHC/2/en.md

```diff
@@ -37,7 +37,7 @@ The Yuyuko's Hidden Cup Season 2 was run by various community members.
 | :-- | :-- |
 | Manager | ::{ flag=CN }:: [SIyuyuko](https://osu.ppy.sh/users/9794030) |
 | Mappool selector | ::{ flag=CN }:: [SIyuyuko](https://osu.ppy.sh/users/9794030), ::{ flag=CN }:: [Muziyami](https://osu.ppy.sh/users/7003013) |
-| Mappool playtester | ::{ flag=CN }:: [Hakumo Shiro](https://osu.ppy.sh/users/5179557), ::{ flag=CN }:: [Hakumo Ai](https://osu.ppy.sh/users/5179557), ::{ flag=CN }:: [VialVistas](https://osu.ppy.sh/users/9697708), ::{ flag=CN }:: [Crystal](https://osu.ppy.sh/users/1646397) ::{ flag=CN }:: [Sayori\_yui](https://osu.ppy.sh/users/7183040) ::{ flag=CN }:: [Senbe1](https://osu.ppy.sh/users/6911753) |
+| Showcaser | ::{ flag=CN }:: [Hakumo Shiro](https://osu.ppy.sh/users/5179557), ::{ flag=CN }:: [Hakumo Ai](https://osu.ppy.sh/users/694480), ::{ flag=CN }:: [VialVistas](https://osu.ppy.sh/users/9697708), ::{ flag=CN }:: [Crystal](https://osu.ppy.sh/users/1646397), ::{ flag=CN }:: [Sayori\_yui](https://osu.ppy.sh/users/7183040), ::{ flag=CN }:: [Senbe1](https://osu.ppy.sh/users/6911753), ::{ flag=CN }:: [carrywind](https://osu.ppy.sh/users/12228018) |
```

Comment:
other tournament articles use "mappool playtester". unless "showcaser" is a completely different role, it would be better to use the more standard term.

Reply:
We confused the concept before, as these players only provided replay files for showcase, did not have any effect in selecting mappool.

Strictly defined, these should truly belong to mappool playtester: Muziyami, SIyuyuko